docker-compose.production.yml

```diff
@@ -0,0 +1,44 @@
+version: '3'
+services:
+  db:
+    image: "postgres:latest"
+    environment:
+      - POSTGRES_USER=${POSTGRES_USER}
+      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
+      - POSTGRES_HOST=${POSTGRES_HOST}
+      - POSTGRES_PORT=${POSTGRES_PORT}
+      - POSTGRES_DB=${POSTGRES_DB}
+    volumes:
+      - ./db-data:/var/lib/postgresql/data
+    networks:
+      - db_nw
+  flaskapp:
+    build:
+      context: .
+      dockerfile: Dockerfile.prod
+    environment:
+      - DATABASE_URL=postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@${POSTGRES_HOST}:${POSTGRES_PORT}/${POSTGRES_DB}
+    networks:
+      - db_nw
+      - web_nw
+    depends_on:
+      - db
+  nginx:
+    image: "nginx:latest"
+    ports:
+      - "80:80"
+      - "443:443"
+    volumes:
+      - ./nginx/flaskapp_prod.conf:/etc/nginx/conf.d/default.conf
+      - ./dh-param/dhparam-2048.pem:/etc/ssl/certs/dhparam-2048.pem
```

Comment:
Where does this key come from? And is it required to already exist on whatever machine this is stood up on for it to work?

Reply:
My understanding is that it sets the parameters for Diffie–Hellman key exchange. It needs to be generated on whatever machine someone intends to run it on. I'll add some info to the README to explain how to create it as well as the let's encrypt stuff.